app/main/createWindow.js

```diff
@@ -41,6 +52,9 @@ export default ({ src, isDev }) => {
   // Setup event listeners for main window
   globalShortcut.register(shortcut, toggleMainWindow)
 
+  // Set the theme vibrancy from config
+  mainWindow.setVibrancy(themeMode(config.get('theme')))
```

Comment:
It'll probably fail silently, but I'll still need to add a class to the body element (or browser window) so it has a solid color instead of being transparent :-)